lib/ddtrace/metrics.rb

```diff
@@ -2,8 +2,8 @@
 
 require 'set'
 require 'logger'
-require 'ddtrace/environment'
-require 'ddtrace/runtime/identity'
+require 'datadog/core/environment/identity'
+require 'ddtrace/ext/environment'
```

Comment:
Nice catch, this was definitely wrong before 👍 

(And on multiple files too 😅 )